crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -0,0 +1,377 @@
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            request_id: Uuid::new_v4().to_string(),
+            amount: item.request.amount,
+            currency: item.request.currency,
+            merchant_order_id: Uuid::new_v4().to_string(),
+        })
+    }
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexPaymentsRequest {
+    request_id: String,
+    payment_method: AirwallexPaymentMethod,
+    payment_method_options: Option<AirwallexPaymentOptions>,
+    return_url: Option<String>,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(untagged)]
+pub enum AirwallexPaymentMethod {
+    Card(AirwallexCard),
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexCard {
+    card: AirwallexCardDetails,
+    #[serde(rename = "type")]
+    payment_method_type: AirwallexPaymentType,
+}
+#[derive(Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexCardDetails {
+    expiry_month: String,
+    expiry_year: String,
+    number: Secret<String, pii::CardNumber>,
+    cvc: String,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum AirwallexPaymentType {
+    Card,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum AirwallexPaymentOptions {
+    Card(AirwallexCardPaymentOptions),
+}
+#[derive(Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexCardPaymentOptions {
+    auto_capture: bool,
+}
+
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for AirwallexPaymentsRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
+        let mut payment_method_options = None;
+        let payment_method = match item.request.payment_method_data.clone() {
+            api::PaymentMethod::Card(ccard) => {
+                payment_method_options =
+                    Some(AirwallexPaymentOptions::Card(AirwallexCardPaymentOptions {
+                        auto_capture: matches!(
+                            item.request.capture_method,
+                            Some(enums::CaptureMethod::Automatic) | None
+                        ),
+                    }));
+                Ok(AirwallexPaymentMethod::Card(AirwallexCard {
+                    card: AirwallexCardDetails {
+                        number: ccard.card_number,
+                        expiry_month: ccard.card_exp_month.peek().to_string(),
+                        expiry_year: ccard.card_exp_year.peek().to_string(),
+                        cvc: ccard.card_cvc.peek().to_string(),
+                    },
+                    payment_method_type: AirwallexPaymentType::Card,
+                }))
+            }
+            _ => Err(errors::ConnectorError::NotImplemented(
+                "Unknown payment method".to_string(),
+            )),
+        }?;
+        Ok(Self {
+            request_id: Uuid::new_v4().to_string(),
```

Comment:
What this `request_id` refers to?

Reply:
Unique ID to be sent for each transaction/operation request to the connector